example-raft-kv/src/network/management.rs

```diff
@@ -29,30 +31,36 @@ pub async fn change_membership(
     app: Data<ExampleApp>,
     req: Json<BTreeSet<NodeId>>,
 ) -> actix_web::Result<impl Responder> {
-    let response = app.raft.change_membership(req.0, true).await;
-    Ok(Json(response))
+    let res = app.raft.change_membership(req.0, true).await;
+    Ok(Json(res))
 }
 
 /// Initialize a single-node cluster.
 #[post("/init")]
 pub async fn init(app: Data<ExampleApp>) -> actix_web::Result<impl Responder> {
     let mut nodes = BTreeSet::new();
     nodes.insert(app.id);
-    let response = app.raft.initialize(nodes).await;
-    Ok(Json(response))
+    let res = app.raft.initialize(nodes).await;
+    Ok(Json(res))
 }
 
 /// Get the latest metrics of the cluster
 #[get("/metrics")]
 pub async fn metrics(app: Data<ExampleApp>) -> actix_web::Result<impl Responder> {
-    let response = app.raft.metrics().borrow().clone();
-    Ok(Json(response))
+    let metrics = app.raft.metrics().borrow().clone();
+
+    let res: Result<RaftMetrics, Infallible> = Ok(metrics);
+    Ok(Json(res))
```

Comment:
As result is infallible, so why not unwrap res?

Reply:
In order to make every API in the same form:D

Reply:
Barely makes sense

Reply:
So is your comment.